modules/era/sql/era_droplist.sql

```diff
@@ -4416,28 +4405,23 @@ INSERT INTO `mob_droplist` VALUES (491,0,0,1000,2171,80);  -- Colibri Beak (8.0%
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5074,@UNCOMMON); -- Scroll Of Maidens Virelai (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,624,@RARE);      -- Clump Of Pamtam Kelp (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,887,@RARE);      -- Coral Fragment (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,4360,@RARE);     -- Bastore Sardine (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,4443,@RARE);     -- Cobalt Jellyfish (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,4514,@RARE);     -- Quus (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5000,@RARE);     -- Scroll Of Knights Minne Iv (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5005,@RARE);     -- Scroll Of Valor Minuet Iv (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5039,@RARE);     -- Scroll Of Herculean Etude (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5040,@RARE);     -- Scroll Of Uncanny Etude (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5073,@RARE);     -- Scroll Of Chocobo Mazurka (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5022,@UNCOMMON); -- Scroll Of Warding Round (Uncommon, 10%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,888,@UNCOMMON);  -- Seashell (Uncommon, 10%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,1435,@RARE);     -- Bards Testimony (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,4981,@RARE);     -- Scroll Of Foe Requiem Vi (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5045,@RARE);     -- Scroll Of Bewitching Etude (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5072,@RARE);     -- Scroll Of Goddesss Hymnus (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5041,@VRARE);    -- Scroll Of Vital Etude (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5042,@VRARE);    -- Scroll Of Swift Etude (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5043,@VRARE);    -- Scroll Of Sage Etude (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5044,@VRARE);    -- Scroll Of Logical Etude (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5074,@VRARE);    -- Scroll Of Maidens Virelai (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,887,@VRARE);     -- Coral Fragment (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5000,@VRARE);    -- Scroll Of Knights Minne Iv (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5005,@VRARE);    -- Scroll Of Valor Minuet Iv (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5039,@VRARE);    -- Scroll Of Herculean Etude (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5040,@VRARE);    -- Scroll Of Uncanny Etude (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (492,0,0,1000,5073,@VRARE);    -- Scroll Of Chocobo Mazurka (Very Rare, 1%)
```

Comment:
http://ffxidb.com/zones/176/coastal-sahagin

I would prob make Logical 5% also if we are keeping them like this. Either way this looks VERY low drop rates compared to ffxidb. We should prob get a capture of this over X amount of kills to see how many can actually drop at once. Something seems very very off.